requirements.txt

```diff
@@ -1,2 +1,3 @@
 docker-py
 dockerfile-parse
+#pygit2
```

Comment:
There's a story here, and I forgot to tell it in the description.

Yes, in theory, we just uncomment that line. The plugin requires pygit2. However, if we do that, Travis CI will try to install it with `pip install`, and that will require building it with gcc which will fail. It will fail even if we install the dev packages it requires.

Of course, we could avoid that by simply installing the Ubuntu packaged version in the `.travis.yml` file... except pygit2 isn't packaged for Ubuntu(!) so that isn't possible after all.

So in short it's commented out because Travis CI's docker container images are based on Ubuntu.